torch/_dynamo/convert_frame.py

```diff
@@ -222,14 +224,21 @@ def format_error_msg(exc, code, record_filename=None, frame=None):
 
 
 def augment_exc_message(exc, msg="\n"):
-    if hasattr(exc, "real_stack") and len(exc.real_stack) > 0 and not config.verbose:
-        msg += f"\nfrom user code:\n {''.join(traceback.format_list([exc.real_stack[-1]]))}"
+    if (
+        hasattr(exc, "real_stack")
+        and len(exc.real_stack) > 0
+        and not (config.verbose and config.suppress_errors)
+    ):
+        msg += f"\nfrom user code:\n {''.join(traceback.format_list(list(reversed(exc.real_stack)))[0:2])}"
```

Comment:
Hmm isn't it gonna be the same as old behaviour now since you are reversing the list?

Reply:
Do we not print the whole thing when the config.verbose is True

Reply:
Yeah you're right about the reversal, long day. Fixed now :) So config.verbose is handled above in format_error_msg. The reason for this is because I only do that formatting if we are logging the exception (we only log the exception if suppress_errors is True)

Reply:
Hhha i feel you. It looks still reversed tho. Are you saying the whole user stacktrace will be present when we do suppress_errors = True?